test/stdlib/YAML_test.rb

```diff
@@ -0,0 +1,16 @@
+require_relative "test_helper"
+require 'yaml'
+
+class YAMLTest < Minitest::Test
+  include TypeAssertions
+  library 'yaml'
+  testing 'singleton(::YAML)'
+
+  def test_load_returns_a_string
+    assert_send_type  "(String) -> (Hash | String | Array)", YAML, :load, "abcde"
+  end
+
+  def test_load_returns_a_hash
+    assert_send_type  "(String) -> (Hash | String | Array)", YAML, :load, "--- \n- Apple \n- Orange "
+  end
```

Comment:
Running `bundle exec rake test TEST=./test/stdlib/YAML_test.rb` 

Results in 
```
Minitest::Assertion: Call trace does not match with given method type: #<struct RBS::Test::CallTrace method_name=:load, method_call=#<RBS::Test::ArgumentsReturn:0x00007ff9856131c8 @arguments=["--- \n- Apple \n- Orange "], @exit_value=["Apple", "Orange"], @exit_type=:return>, block_calls=[], block_given=false>.
Expected ["[Moduleload] ReturnTypeError: expected `Hash | String | Array` but returns `[\"Apple\", \"Orange\"]`"] to be empty.
```

Is there something wrong with the definition of the return signature ?

Reply:
Arrays should be declared `Array[untyped]` and Hash `Hash[untyped, untyped]`